From 16013ff37d07e26919816215d87f97e24ab5c490 Mon Sep 17 00:00:00 2001
From: Andrew Lunn <andrew@lunn.ch>
Date: Mon, 5 May 2025 18:39:12 -0500
Subject: [PATCH 14/30] net: phy: maxio: Delete broken message about strapping

phy_write() success is success, nothing more. It is not possible to
tell from the return value if it did not change anything. So delete
this bogus message.

Signed-off-by: Andrew Lunn <andrew@lunn.ch>
---
 drivers/net/phy/maxio.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/net/phy/maxio.c b/drivers/net/phy/maxio.c
index f1f1e789d..364eb6b13 100755
--- a/drivers/net/phy/maxio.c
+++ b/drivers/net/phy/maxio.c
@@ -112,10 +112,6 @@ static int maxio_mae0621a_config_init(struct phy_device *phydev)
 	if (ret < 0) {
 		dev_err(dev, "Failed to update the TX delay register\n");
 		return ret;
-	} else if (ret == 0) {
-		dev_dbg(dev,
-			"2ns  delay was already %s (by pin-strapping RXD1 or bootloader configuration)\n",
-			val ? "enabled" : "disabled");
 	}
 delay_skip:
 
-- 
2.39.5

